website/content/docs/discovery/dns.mdx

```diff
@@ -96,6 +96,41 @@ pairs according to [RFC1464](https://www.ietf.org/rfc/rfc1464.txt).
 Alternatively, the TXT record will only include the node's metadata value when the
 node's metadata key starts with `rfc1035-`.
 
+## Alternative Domain
+
+By default, Consul responds to DNS queries only for its configured
+[`domain`](/docs/agent/options#domain).
+
+Some use cases require responding to queries for more than one domain,
+such as during a DNS migration or to distinguish between internal and
+external queries by using different domains.
+
+Consul can be configured to respond to DNS queries on an alternative domain
+through the [`alt_domain`](/docs/agent/options#alt_domain) agent configuration
+option. Consul's DNS response will use the same domain as was used in the query.
+For example, if `test-domain` is configured as the alternative domain, the following query:
+
+```shell-session
+$ dig @127.0.0.1 -p 8600  consul.service.test-domain SRV
+```
+will return:
+
+```
+;; QUESTION SECTION:
+;consul.service.test-domain.	IN	SRV
+
+;; ANSWER SECTION:
+consul.service.test-domain. 0	IN	SRV	1 1 8300 machine.node.dc1.test-domain.
+
+;; ADDITIONAL SECTION:
+machine.node.dc1.test-domain. 0	IN	A	127.0.0.1
+machine.node.dc1.test-domain. 0	IN	TXT	"consul-network-segment="
+```
+
+-> **Note:** Responses to PTR queries (`<ip>.in-addr.arpa.`) will always use the
```

Comment:
```suggestion
-> **PTR queries:**  Responses to PTR queries (`<ip>.in-addr.arpa.`) will always use the
```
I think it would be a good idea to start using more meaningful labels in these callout blocks. Also, is a PTR query a pointer query? If so, we should say "Responses to pointer (PTR) queries . . ."